docs/resources.md

```diff
@@ -584,89 +584,7 @@ spec:
 
 ### Image Resource
 
-An `image` resource represents an image that lives in a remote repository. It is
-usually used as [a `Task` `output`](tasks.md#outputs) for `Tasks` that build
-images. This allows the same `Tasks` to be used to generically push to any
-registry.
-
-Params that can be added are the following:
-
-1.  `url`: The complete path to the image, including the registry and the image
-    tag
-1.  `digest`: The
-    [image digest](https://success.docker.com/article/images-tagging-vs-digests)
-    which uniquely identifies a particular build of an image with a particular
-    tag. _While this can be provided as a parameter, there is not yet a way to
-    update this value after an image is built, but this is planned in
-    [#216](https://github.com/tektoncd/pipeline/issues/216)._
-
-For example:
-
-```yaml
-apiVersion: tekton.dev/v1alpha1
-kind: PipelineResource
-metadata:
-  name: kritis-resources-image
-  namespace: default
-spec:
-  type: image
-  params:
-    - name: url
-      value: gcr.io/staging-images/kritis
-```
-
-#### Surfacing the image digest built in a task
-
-To surface the image digest in the output of the `taskRun` the builder tool
-should produce this information in a
-[OCI Image Layout](https://github.com/opencontainers/image-spec/blob/master/image-layout.md)
-`index.json` file. This file should be placed on a location as specified in the
-task definition under the default resource directory, or the specified
-`targetPath`. If there is only one image in the `index.json` file, the digest of
-that image is exported; otherwise, the digest of the whole image index would be
-exported. For example this build-push task defines the `outputImageDir` for the
-`builtImage` resource in `/workspace/buildImage`
-
-```yaml
-apiVersion: tekton.dev/v1beta1
-kind: Task
-metadata:
-  name: build-push
-spec:
-  resources:
-    inputs:
-      - name: workspace
-        type: git
-    outputs:
-      - name: builtImage
-        type: image
-        targetPath: /workspace/builtImage
-  steps: ...
-```
-
-If no value is specified for `targetPath`, it will default to
-`/workspace/output/{resource-name}`.
-
-_Please check the builder tool used on how to pass this path to create the
-output file._
-
-The `taskRun` will include the image digest and URL in the `resourcesResult` field that
-is part of the `taskRun.Status`
-
-for example:
-
-```yaml
-status:
-    # ...
-    resourcesResult:
-      - key: "digest"
-        value: "sha256:eed29cd0b6feeb1a92bc3c4f977fd203c63b376a638731c88cacefe3adb1c660"
-        resourceName: skaffold-image-leeroy-web
-    # ...
-```
-
-If the `index.json` file is not produced, the image digest will not be included
-in the `taskRun` output.
+Image Resource is deprecated and please refer to the doc at https://github.com/tektoncd/pipeline/blob/main/docs/pipelineresources.md#replacing-an-image-resource for migration.
```

Comment:
```suggestion
Support for image resources has been removed. Please refer to the doc at https://github.com/tektoncd/pipeline/blob/main/docs/pipelineresources.md#replacing-an-image-resource for migration.
```

suggesting since "deprecation" implies that it still works, while "removal" implies that it does not.